src/libs/ReportUtils.ts

```diff
@@ -781,7 +781,7 @@ function hasParticipantInArray(report: OnyxEntry<Report>, memberAccountIDs: numb
 }
 
 /**
- * Whether the Money Request report is settled
+ * Whether the Expense report is settled
```

Comment:
I kinda feel like we can keep using the money request terminology when its both expense and IOU as we need some collective word
```suggestion
 * Whether the Money Request report is settled
```